src/panoramic/cli/local/reader.py

```diff
@@ -67,6 +74,11 @@ def get_packages(self) -> Iterable[FilePackage]:
                 name=d.name,
                 data_source_file=d / PresetFileName.DATASET_YAML.value,
                 model_files=list(d.glob(f'*{FileExtension.MODEL_YAML.value}')),
+                field_files=list(d.glob(f'**/*{FileExtension.FIELD_YAML.value}')),
```

Comment:
Shouldn't this be looking at `{SystemDirectory.FIELDS.value}/*{FileExtension.FIELD_YAML.value}` only?

Reply:
Done in 6791f903757b3015b22c3f46a46a099f39214499.